app/components/network-message.tsx

```diff
@@ -1,26 +1,28 @@
-import React from 'react';
-import { Flex, Text, Box } from '@blockstack/ui';
+import React, { FC } from 'react';
+import { Text, Box } from '@blockstack/ui';
 
-import { ENV } from '../constants/index';
+// import { ENV } from '@constants/index';
```

Comment:
Wondering why you removed it - simplicity? Or, do we need a separate `ENV`-type variable, i.e. `testnet` vs `mainnet`